packages/cli/src/controllers/owner.controller.ts

```diff
@@ -33,16 +34,18 @@ export class OwnerController {
 		logger,
 		internalHooks,
 		repositories,
+		userService,
 	}: {
 		config: Config;
 		logger: ILogger;
 		internalHooks: IInternalHooksClass;
-		repositories: Pick<IDatabaseCollections, 'User' | 'Settings'>;
+		repositories: Pick<IDatabaseCollections, 'Settings'>;
```

Comment:
we should also switch away from injecting a `repositories` object like this, as that isn't compatible with dependency injection. we should switch over to passing each individual repository class instead.